cmd/hook/main.go

```diff
@@ -72,6 +73,10 @@ type options struct {
 	instrumentationOptions prowflagutil.InstrumentationOptions
 	jira                   prowflagutil.JiraOptions
 
+	enableSSL bool
+	certFile  string
+	keyFile   string
```

Comment:
let's group this into a small struct that we can embed at both places, we can share validation etc. similarly to how the `ThingOptions` bits above